### PR TITLE
Angular2

### PR DESCRIPTION
**Pull request by magarcia**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -1,74 +1,173 @@
...
+  Git](https://help.github.com/articles/set-up-git) is a good source of information.
+
+* [Node.js](http://nodejs.org), (version `>=5.4.1 <6`) which is used to run a
+  development web server, run tests, and generate distributable files. We also
+  use Node's Package Manager, `npm` (version `>=3.5.3 <4.0`), which comes with
+  Node. Depending on your system, you can install Node either from source or as
+  a pre-packaged bundle.
+
+* [Janus-Gateway](http://janus.conf.meetecho.com/) is the server-side WebRTC
+  handling, so is necessary a running janus server with support fro data
```
Comment (imobach) [inline DEVELOPMENT.md]: typo: `fro` -> `for`. Also I would upcase `janus`.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -1,74 +1,173 @@
...
 
-Then you could adapt the configuration by creating a file called
-`src/config.json`. You have an sample file in `src/config.json.sample`.
+## Installing Janus
 
-For a development environment, the next configuration would be fine:
+For most Linux distributions that
+translates into installing the `janus` or `janus-gateway` package.
+Make sure those packages include data channels support. Compiling Janus Gateway
+from the sources is, of course, also an option.
```
Comment (imobach) [inline DEVELOPMENT.md]: I would omit `the`.

File: config/spec-bundle.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+/*
+ * When testing with webpack and ES6, we have to do some extra
```
Comment (imobach) [inline config/spec-bundle.js]: Are we going to use ES6 or TypeScript for the tests?

File: src/app/components/browser-info/browser-info.controller.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+BrowserInfoCtrl.$inject = ["$uibModalInstance", "RequestService"];
```
Comment (imobach) [inline src/app/components/browser-info/browser-info.controller.ts]: No more ```(function () {...})()```... nice :)

**Comment by imobach** [comment]
Wow! It's an amazing amount of work. Thanks for that!

The code looks good, but I want to try it by myself following your documentation. I expect to do it today or tomorrow.

**Comment by magarcia** [comment]
@imobach thanks for the review. I just made a commit solves these changes

**Comment by imobach** [comment]
LGTM. I could run the new setup in an AWS machine. I'll merge the changes in the new `angular2`.

Thanks!

**Pull request merged**